Breeding-Insight/bi-web
# Feature/bi 1459

# Description
**Story:** https://breedinginsight.atlassian.net/browse/BI-1459

Created a page to view the status of asynchronous jobs that have been submitted for a given program.  Updated the job submission pages (germplasm import, brapi import) to show a message after a job is submitted telling the user they can navigate away from the page, and view the status of the job on the new job status page.

Also removed some of the clutter on the welcome screen for now until we decide what content we want on the welcome screen.



# Dependencies
bi-api: [features/BI-1459](https://github.com/Breeding-Insight/bi-api/tree/feature/BI-1459)

# Testing
*Job status does not show for ontology upload, as that does not use the import tables to store the status of the upload.*

- Upload a file via the germplasm uploader, or the BrAPI import tool
- Confirm the import was submitted successfully
- A `warning` styled message should be displayed at the top of the import page indicating the job has been submitted, and status can be checked on the jobs page 
- Navigate to the jobs page (should be a menu item for this)
- Verify the job that was just submitted displays in the list of jobs
- Refresh the page until the job status completes (either successfully or in error)
- If successful, there should be a green pill for the row with the word "Finished" 
- If failed, there should be a red pill for the row with the word "Error"


# Checklist:

- [ ] ~I have tested that my code works with both the brapi-java-server and BreedBase~ NA
- [ ] I have run TAF: https://github.com/Breeding-Insight/taf/actions/runs/2423334177
